src/it/scala/BulkLoadData.scala

```diff
@@ -94,13 +98,37 @@ object BulkLoadData {
     "http://localhost:8080/rdf4j-server/repositories/PACT/statements"
   )
 
+  private val postRecordConceptData = Seq(
```

Comment:
Are we invoking `curl` from Scala? That seems like an unnecessary thing to do, as we could just use a http library to POST the content instead. That need not stop this PR though - we could address that in a piece of future technical debt perhaps...